aptos-move/framework/aptos-framework/sources/genesis.move

```diff
@@ -110,48 +122,49 @@ module aptos_framework::genesis {
     ///
     /// Network address fields are a vector per account, where each entry is a vector of addresses
     /// encoded in a single BCS byte array.
-    fun create_initialize_validators(
-        aptos_framework_account: signer,
-        owners: vector<address>,
-        consensus_pubkeys: vector<vector<u8>>,
-        proof_of_possession: vector<vector<u8>>,
-        validator_network_addresses: vector<vector<u8>>,
-        full_node_network_addresses: vector<vector<u8>>,
-        staking_distribution: vector<u64>,
-    ) {
-        let num_owners = vector::length(&owners);
-        let num_validator_network_addresses = vector::length(&validator_network_addresses);
-        let num_full_node_network_addresses = vector::length(&full_node_network_addresses);
-        assert!(num_validator_network_addresses == num_full_node_network_addresses, 0);
-        let num_staking = vector::length(&staking_distribution);
-        assert!(num_full_node_network_addresses == num_staking, 0);
+    fun create_initialize_validators(aptos_framework_account: signer, validators_serialized: vector<u8>) {
```

Comment:
oh jeez dude.... just define 

vec<ValidatorConfiguration> and play with it until you get it right...

Reply:
lolll but why? The struct is so much cleaner :)

Reply:
how is this cleaner??

you can still pass in `validators: vec<ValidatorConfiguration>`

Reply:
Is that possible without much more work to convert from the Rust struct to MoveValue::Struct fields?